cpu/kinetis/include/periph_cpu.h

```diff
@@ -202,6 +226,7 @@ enum {
  * @{
  */
 #define HAVE_ADC_RES_T
+#ifdef KINETIS_HAVE_ADC_16BIT
```

Comment:
why?

Reply:
Ow, sorry, that might be related to changes to adc, which I didn't test and decided not to include into this PR. I'll remove that.

Reply:
@smlng Actually this `#ifdef` still makes sense. `ADC_CFG1_MODE` etc. are not defined for all CPUs. What's important here, it's not defined for S9KEA.

Reply:
mhm, the name of the macro suggests that the `ifdef` should only change the definition of `ADC_RES_16BIT` and not including or excluding the whole ADC config.

Reply:
@smlng It _might_ be a misnomer, yes. The point is that KEA ADC (not in scope of this PR) has a different set of registers than the one already implemented here. Since they don't have any special names and only differ in resolution, I've called the defines `KINETIS_HAVE_ADC_16BIT` and `KINETIS_HAVE_ADC_12BIT` (the latter is, again, not included here). I could rename it to something more meaningful (like e.g. `KINETIS_HAVE_KEA_ADC` and `KINETIS_HAVE_K_ADC`).